### PR TITLE
Briancoughlin patch 2

### PR DESCRIPTION
**Pull request by Briancoughlin**
Suggested updates to the readme based on information gained writing the getting-started guide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,17 @@
 
 | 🛑  IMPORTANT - Early Access 🛑  | 
 | -- |
-| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package. The MLAPI package  is on the road to being a fully featured solution. We have solutions architects available on Discord and forums to help you work through issues you encounter. |
+| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package.
```
Comment (lkrell) [inline README.md:6]: need closing table
```suggestion
| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package.  |
```

File: README.md [hunk 2]
```diff
@@ -3,12 +3,17 @@
...
-| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package. The MLAPI package  is on the road to being a fully featured solution. We have solutions architects available on Discord and forums to help you work through issues you encounter. |
+| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package. 
 
 Boss Room is a fully functional co-op multiplayer RPG made with Unity MLAPI. It is built to serve as an educational sample that showcases certain typical gameplay patterns that are frequently featured in similar games.
 
 Our intention is that you can use everything in this project as a starting point or as bits and pieces in your own Unity games. The project is licensed under the Unity Companion License. See [LICENSE.md](LICENSE.md) for more legal information.
 
+> __IMPORTANT__:
+> - Boss Room supports all major Unity platforms. To use the WebGL platform a custom WebGL transport based on web sockets is needed.
+> - Boss Room is compatible with Unity 2020 LTS onwards.
```
Comment (lkrell) [inline README.md:14]:
```suggestion
> - Boss Room is compatible with Unity 2020 LTS and later.
```

File: README.md [hunk 3]
```diff
@@ -67,14 +72,14 @@ Running the game over internet currently requires setting up a [Photon Transport
 > Checkout our Photon-Realtime setup guide, here:
 > [Boss Room Photon Setup Guide](Documentation/Photon-Realtime/Readme.md)
 
-Alternatively you can use Port Forwarding. The wonderful https://portforward.com/ site has guides on how to enable port forwarding on a huge number of routers. Boss Room uses `UDP` and needs a `9998` external port to be open. 
+Alternatively you can use Port Forwarding. The https://portforward.com/ site has guides on how to enable port forwarding on a huge number of routers. Boss Room uses `UDP` and needs an `9998` external port to be open.
```
Comment (lkrell) [inline README.md:75]: a `9998` is correct
```suggestion
Alternatively you can use Port Forwarding. The https://portforward.com/ site has guides on how to enable port forwarding on a huge number of routers. Boss Room uses `UDP` and needs a `9998` external port to be open. 
```

**Review by lkrell — COMMENTED** [review]
Quick feedback

**Comment by SamuelBellomo** [comment]
This should be merged after your other fix is merged

File: README.md (hunk 3 above)
Comment (Briancoughlin) [reply README.md:75]: we had a lot of thoughts on 'a' or 'an' do you ignore the numbers  or not ....im not sure

**Comment by Briancoughlin** [comment]
have added the changes for the other PR ( Github was mucking about earlier so it lost my initial PR so i redid it ). everything should be in this one now.

File: README.md (hunk 1 above)
Comment (Briancoughlin) [reply README.md]: old text returned as per Sams query.

File: README.md (hunk 2 above)
Comment (Briancoughlin) [reply README.md]: Fix added in edited file

File: README.md [hunk 4]
```diff
@@ -9,6 +9,11 @@ Boss Room is a fully functional co-op multiplayer RPG made with Unity MLAPI. It
 
 Our intention is that you can use everything in this project as a starting point or as bits and pieces in your own Unity games. The project is licensed under the Unity Companion License. See [LICENSE.md](LICENSE.md) for more legal information.
 
+> __IMPORTANT__:
+> - Boss Room supports those platforms supported by MLAPI (Windows and Mac). To use the WebGL platform a custom WebGL transport based on web sockets is needed.
```
Comment (SamuelBellomo) [inline README.md:13]: Not sure I would mention WebGL, there was zero dev on this

Comment (Briancoughlin) [reply README.md:13]: that's the text for MLAPI, was there work on MLAPI on it?

Comment (Briancoughlin) [reply README.md:13]: its your readme, you can remove this if you think its not valid

Comment (Briancoughlin) [reply README.md]: Edited file

**Review by SamuelBellomo — APPROVED** [review]

**Review by pdeschain — APPROVED** [review]

**Review by lkrell — APPROVED** [review]

**Pull request merged**